real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/serial/RBSerialMessage.java

```diff
@@ -14,6 +14,7 @@
 	public static final byte AUTO = (byte)22;
 	public static final byte BATTERY = (byte)23;
 	public static final byte RBSM_MID_MEGA_STEER_FEEDBACK = (byte)24; //potentiometer 
+	public static final byte RBSM_MID_MEGA_STEER_ANGLE = (byte)20;
```

Comment:
Why did we make a new definition for this? There's already a definition for steering on line 12.